### ISSUE TITLE
Community policy on listing resources that are plagiarised and violate the CC licence?

### ISSUE THREAD

**Issue by svaksha** [JuliaLang/julialang.github.com#194]
Ref: https://github.com/JuliaLang/julialang.github.com/pull/192

With reference to the PR #192 that was merged today, does the Julia community have a policy on listing duplicate resources that involve plagiarism and also breaking the original CC licence by changing it to Apache? 

A quick comparision of `Julia.jl` with `awesome-julia` [1] by @melvin0008 tells me that less than a dozen listing of resources are different. TBH, I saw 'awesome-julia' some days ago and ignored it as I didnt want to kick up a fuss, but after seeing his PR, I realize I should not have ignored the plagiarism. It would have been nicer if @melvin0008) would have submitted a PR to `Julia.jl` for any links that he felt were missing - I've had folks submit PR's and merged all those who wanted to contribute.

Unfortunately, instead of contributing and collaborating, he (melvin) has chosen to break the license terms and indulge in blatant plagiarism - Since last week he has forked `Julia.jl` (I have a pic if you want proof), blatantly copied more resources (that I have spent years collecting) into his repo, broken the CC terms under which I publicly share the repo by changing the licence to Apache inorder to get himself listed on the Julialang website. JFTR, the CC-BY-NC-ND allows a licence change but not the CC-BY-NC-SA 4.0 (International), and I have intentionally chosen the latter licence to minimize/avoid such problems. 

On a relevant note, I must mention that last month Stefan had emailed me asking if they could use Julia.jl as the basis for forming a list of unregistered packages and I was thrilled that it would be considered good enough. Fwiw, the core-devs didnt need my permission but I appreciate the fact that they are polite enough to ask. That said, I have no issues with folks who actually compete and create another awesome-* meme .. heck, I do too by curating a scientific Python resource repo (pythonidae) which is vastly different from `awesome-python` by @vinta and `pycrumbs`. 

@melvin0008, copying my (and other contributors) work by rearranging them on a single README page and changing the licence does not change the fact that you are disrespectful of other peoples efforts, have broken the CC license terms and by getting yourself listed on the Julialang website, the project has (unknowingly) provided legitimacy to someone who is has knowingly broken the Licence terms. That isnt how FOSS works, where breaking licences and/or plagiarism is taken seriously. 

The same situation can arise with packages too, so I hope the Julia community responds pro-actively and also formulates some guidelines (or policy) while responding and dealing with Foss licencing and plagiarism issues in future.

Thanks for reading!
Best, 
- svaksha.

[1] The name `awesome-julia` isnt new btw. https://github.com/dantedurrman/awesome-julia, was started last year and the owner deleted it of his own accord instead of duplicating `Julia.jl`, which I appreciate than the licence change without permission and blatant plagiarism.

**Comment by svaksha** [comment]
TBH, I didnt expect a defense of plagiarism and license violation from
a core-dev so my reply is inline below :

On Fri, Jan 9, 2015 at 11:10 AM, Viral B. Shah <notifications@github.com> wrote:
>
> FWIW, I had suggested to Melvin to list it on the Julia website in JuliaLang/julia#9667
>
> Have you written to Melvin with your concerns? If it is derived work, I am sure he will follow the license guidelines and attribution. Only when a person otherwise ignores all communication, I would label it plagiarism.
>

See the commit linked below which proves he has copied an entire
subsection from Julia.jl.
No, I prefer a public discussion - either on the mailing list or a bug
report. Fwiw Viral, in situations like these my experience in going
into private discussions with Indian men has always resulted in either
sexism, more harassment and online stalking, so I avoid it unless I
know the person or have met them and trust them to not be a jerk. Tons
of literature has been written over the years about the state of women
and dicersity in STEM and FOSS communities and I used to blog about my
personal experiences under the category "STEM/DiversityFail" but it
does not list the harassment and sexism that occurred in the Indian
python community in 2013. Anyway, I dont want to digress - the curious
can always read-up on the subject of women in Foss.

> Approaching this positively and respectfully will gain us more contributors, and lead towards the larger improvement of Julia.jl. I have some other thoughts on improving Julia.jl - which I will file as an issue there.
>

Please do. I would love to get feedback on some packages whose
boundaries are not very clear. That said, I was ok with
`awesome-julia` until I saw the verbatim copying. I didnt expect that
at all. It would have been nicer if he had chosen to collaborate like
he did with other asesome-* lists.


> It is perfectly fine for someone to take pieces of other work and combine them in new and interesting ways. That is a fundamental basis of open source. We work hard on avoiding fragments, and often people are happy to collaborate. In some cases, the forks are meaningful and even invited.
>

TBH, I'm not sure what you mean by "the forks are meaningful and even
invited" - is that supposed to justify plagiarism? Let me give you one
such commit example which is a blatant copy:
https://github.com/melvin0008/awesome-julia/commit/ad829e1022811c24749814d4a99a1b5ed92dc60a,
where Melvin has only slightly changed the alphabetical listing I
follow and not even bothered to change the markdown (-) while adding
the + sign. The entire IDE paragraph (verbatim wording) remains what
is already listed at :
https://github.com/svaksha/Julia.jl/blob/master/Utilities.md#ide

Would you still call this verbatim copying a meaningful fork? Viral, I
could probably go into each commit and pick out more examples but that
is pointless when one can see the obvious - its not very hard to make
minor tweaks and change some words here and there. That is missing the
point entirely and very disrespectful of other peoples work - whatever
stopped him from submitting a PR to Julia.jl because if he can respect
a man's work and submit a PR to `awesome-python`
(https://github.com/vinta/awesome-python/commit/5126afd20ce9b7e3a637fb48bea6aa3e3ecc6c1a)
and `awesome-php`, why not do the same with `Julia.jl`? Why diss a
woman's work by copying chunks of it and worse change the CC license
to Apache? That is sexism and its a common attitude in the Indian FOSS
community, so I am very disappointed to hear you justifying plagiarism
and the breaking of a CC license instead of encouraging him to
collaborate and create more meaningful work that benefits the
community. After almost a decade of using Foss, I think I grok the
"fundamental basis of open source" so the patronizing tone isnt
helping the discussion.

-SVAKSHA ॥  http://about.me/svaksha  ॥

**Comment by ViralBShah** [comment]
I feel like I am missing something about the CC license. Doesn't it allow sharing and remixing the content so long as the license conditions are followed? In my understanding, this would be completely ok, so long as the terms of the license are followed. If I am missing something, I would love to be corrected. I have myself released content under CC, with the expectation that it will be copied, modified, and shared.

In this case, Melvin should attribute the original work, and require further derived work to follow the same license. If that is done, does it not resolve the issue at hand?

Also, there are far too many jibes against various types of people in your comment above - which are personal views and not helping the discussion here, and strictly not civil. I am requesting you edit the post above accordingly.

**Comment by svaksha** [comment]
Replying inline:

On Fri, Jan 9, 2015 at 12:51 PM, Viral B. Shah <notifications@github.com> wrote:
>
> I feel like I am missing something about the CC license. Doesn't it allow sharing and remixing the content so long as the license conditions are followed? In my understanding, this would be completely ok, so long as the terms of the license are followed. If I am missing something, I would love to be corrected. I have myself released content under CC, with the expectation that it will be copied, modified, and shared.
>

Not really - there is a huge difference in context, and especially
content that you license under CC .... see my comment about the
difference between 2 CC licences
(https://github.com/chrisvoncsefalvay/learn-julia-the-hard-way/issues/15#issuecomment-68167187)
when you talk of remixing or sharing. IANAL, but the two (apache and
CC) are incompatible licenses, see the link/quote and reply below.

> In this case, Melvin should attribute the original work, and require further derived work to follow the same license. If that is done, does it not resolve the issue at hand?
>

From, http://creativecommons.org/compatiblelicenses

BY-NC-SA
Version 4.0
Your contributions to adaptations of material under BY-NC-SA 4.0 may
only be licensed under:

BY-NC-SA 4.0, or a later version of the BY-NC-SA license.
Ported versions of the BY-NC-SA license, version 4.0 or later.
A license designated as a “BY-NC-SA Compatible License” as defined in
BY-NC-SA 4.0.
Currently, no non-CC licenses have been designated as compatible with
BY-NC-SA 4.0.
</quote>

So to answer your question, he (or anyone else) can fork, distribute
(share-alike) and contribute under the terms of the same CC license
(BY-NC-SA-4.0) BUT a problem arose when he didnt, rather, he has
copied chunks of sections and changed the license, which is breaking
the license terms, hence plagiarism.  That is what forced me to file
this bug report in order to understand what the community policy is
going to be about such situations.

> Also, there are far too many jibes against various types of people in your comment above - which are personal views and not helping the discussion here, and strictly not civil. I am requesting you edit the post above accordingly.
>

Best case, I'll try to ignore intent and hope we can agree to disagree
and drop this line of conversation but I cannot deny my negative
experiences within the Indian Foss community, nor change what I
experienced - That would be lying to myself. If this isnt what you
meant, please be specific about which comment in my BR is not civil,
so that I can understand the real issue at hand(?) because my replies
are mostly around the discussion of a license violation (hence
plagiarism) while trying to understand your first response which came
off as defending the violation while telling me I didnt grok Foss. To
me that seemed uncivil as it wasnt helping the discussion, which I
hope stays around the subject line.

As I mentioned in the original BR, competition is certainly welcome,
is the keystone of Foss, and fun (if I may call it that). However,
there is a huge chasm between collaboration and copying. When there
exists an option for the former (in the form of PR's), I am curious to
know why the latter (license change, hence plagiarism) was done and
what the community policy on this will be when you officially list it
on the Julia website?

Tbh, I had no clue that you asked him to list himself on the website,
and the BR was not meant to question your authority either.  Again,
IANAL, but because the CC and Apache are incompatible the attribution
wont change the CC license violation and now that folks are aware of
the CC license violation issue, I hope it explains why the Julia
project has (unknowingly) provided legitimacy to someone breaking the
CC License terms. TBH, I would still prefer a collaboration and
request that Melvin (and anyone else) submit PR's for any links that
are missing in `Julia.jl`, but I dont want to force anyone to do so.

Hope that explains!

@SVAKSHA ॥  http://about.me/svaksha  ॥

**Comment by melvin0008** [comment]
Hey Svaksha,

By no intentions have I tried to take credit for your work.

My main motivation for awesome-julia is to help people get all the resources to start learning about it. Your Repository was focused mainly on listing packages. I have also added your link under packages. I was writing a list of resources for Julia as there is no repository with a prefix of awesome. Also, I had started this with an inspiration to start contributing to Julia as I am studying the Julia code and will soon start contributing to it.

I have just copied the IDE part of your Julia.jl repository to list the available IDE's for developers. If you would like me to remove the IDE section please let me know. I had no intentions in changing CC to Apache. I had just selected Apache without even thinking about it.

BTW I was of the thought that collecting resources, documenting and contributing to the open source community was something out of personal interest and betterment of the developers community. I did not know this would create a fuss and let you berate me on what sexism is. [How is this even remotely related to the issue at hand?]

Regards,
Melvin

**Comment by ihnorton** [comment]
I haven't read the links in question yet, but: could everyone please make an effort to stay on the main issue, which is the copyright question.

I don't think it is the goal of the community to promote copyright violations if such exists. At the same time, I would hope that we can give the benefit of the doubt and try to understand and educate on the requirements of each licensing scheme. This is a very important topic for the continued strength of open-source software, but it also takes time to understand the details. My own (very limited) understanding is that CC-BY-SA can be considered "like the GPL for documentation", and as such any direct copying means that the same license should be applied (as well as attribution provided).

**Comment by melvin0008** [comment]
I will do the following .
i.e change the license and add an attribution. 
I would like to reiterate that I had no intentions in taking credit for your work . I have very less knowledge about licenses and hence would like to state that changing CC to Apache was something out of promptness. 

@svaksha Does changing the license and attributing you to the IDE part resolve the issue.

Please let me know ?

**Comment by one-more-minute** [comment]
To add to what @ihnorton has said, while many of us have years of experience working with FOSS, others don't – and the ins and outs of etiquette and licensing aren't always self-evident. Mistakes will be made even with the best of intentions, but as long as people are willing to fix them and learn (and it looks like that's the case here), no harm done.

**Comment by melvin0008** [comment]
@svaksha I have added an attribution to the IDE paragraph and changed my License to Creative Commons. If there is anything else which needs to be done please let me know

**Comment by melvin0008** [comment]
@svaksha  changed the much debated IDE [link](https://github.com/melvin0008/awesome-julia#ide) part. Thanks anyway. :+1:

**Comment by svaksha** [comment]
On Fri, Jan 9, 2015 at 3:03 PM, Melvin <notifications@github.com> wrote:
>
> I will do the following .
> i.e change the license and add an attribution.
> I would like to reiterate that I had no intentions in taking credit for your work . I have very less knowledge about licenses and hence would like to state that changing CC to Apache was something out of promptness.
>

Usually, I always assume good intent, but your responses in this
thread and github actions have changed that line of thought. All of
these licenses include the legal verbiage (either on the site or
within the repos) so please take the time to read and understand them
and ask first before copying content. As for intent, let me quote
"Primary or secondary evaluations of intent only concern the user
(licensee), not the copyright owner (licensor)" {from,
http://zookeys.pensoft.net/articles.php?id=3036}, which behooves me to
wonder about the intent when you break the license terms and react as
you did when called out. And see below.

AFAIK, one cannot compare a CC license with a software license where a
codebase fork occurs when the new devel plans on taking the project in
a diametrically different direction than what the first author (or
existing core-devs) originally intended because, hypothetically
speaking, his/her patches and PR's didnt align with the original and
existing core-deveopment criteria. That isnt the case here. Besides,
I've explicitly suggested the PR contributions are welcome which you
ignored, so even if I presume good intent, the copied
text/modifications dont offer a great learning opportunity from any
angle as they were already in `Julia.jl`. I am trying very hard to
understand the intent of not preserving the integrity of the original
project and copying content from it because the evaluation of intent
only concerns the user (here, you the "licensee"), whose CR and CC
terms you continue to break.


> @svaksha Does changing the license and attributing you to the IDE part resolve the issue.
>

No, it does not (see above and below). But, before I delve into the
license bits, would you care to explain why you insist on maintaining
a separate resource (which is nothing more than plagiarised content)
instead of collaborating via PR's to Julia.jl? If you had no trouble
in collaborating with other (male) owners of similar lists (viz.
awesome-php and awesome-python), please explain why you are ignoring
my repeated requests to collaborate on `Julia.jl` and choose to break
the CR and CC license instead? This is the first time I am
experiencing a CR and CC license violation in FOSS so I am learning a
lot from this and even the most generous consideration and
interpretation of your claim to want to contribute, does not explain
how 29 people (its 30 forks, including you) who have forked the
`Julia.jl` repo can respect the CR and CC license while you didnt.
Thusfar, in all your responses I didnt see a rationale for any of
these queries.

As for the CC legalese, IANAL and no two licenses (even among the CC)
are the same and it all boils down to the respective licence
interpretation on a case-to-case basis. That said, here are the
relevant important points to be considered in the violation :

1. As per the Section 3(a)(1)(A) of the CC-BY-NC-SA-4.0,
http://creativecommons.org/licenses/by-nc-sa/4.0/legalcode#s3a1A,
attribution for the ShareAlike Licensed Material (including in
modified form) must retain the CR notice, the original license notice,
URL etc.. That means, its perfectly fine if you make those changes and
added content to the `Julia.jl` fork but you didnt. The violation
problem arises because you have forked my repo (which is fine), but
when you create a new repo with a new name and rearranged large parts
of the `Julia.jl` content in this new repo, it creates the impression
of new original work which `awesome-julia` isnt. Also, it cannot be
called an adaptation (a term usually used for translations of original
works) as it contains plagiarised content in the same language
(English) it was released under.

2. IMHO, the optimal solution is the community approach - contribute
all missing links via PR's to `Julia.jl` because as it stands the
`awesome-julia` repo still has content and many links that were first
listed in `Julia.jl` under the CC-BY-NC-SA-4.0, and many of your
commits include links from Julia.jl in a renamed repo which again
violates the CC terms, see :
<https://github.com/melvin0008/awesome-julia/commit/eb9ac4e4c5e184cf8a9f94f74adbe8c8dae97fca>.
I've only linked one commit here but there are plenty more that I have
not listed. My not listing each an every link and content you copied
does not invalidate the violation claim. Usually, I've observed people
respect the licence terms when alerted to it but I'm dismayed to see
that you are adding more of the 'Julia.jl' content into
'awesome-julia'.

3. Furthermore, do note that the CC-BY-NC-SA-4.0 terms also requires
you to "indicate if You modified the Licensed Material and retain an
indication of any previous modifications; and indicate the Licensed
Material is licensed under this Public License, and include the text
of, or the URI or hyperlink to, this Public License." </quote>. But,
by continuing to copy the content (URL and text) from `Julia.jl` into
a new repo that does not use the same name I released it under, you
are misleading folks into thinking it is original work or an
adaptation, which it isnt. JFTR, I didnt waive away my copyrights (I
cant as per the Indian law) nor give permission to rename the repo
(nor create a new repo) while continuing to copy the CC content that I
(and others have) created - hence, copying the URL content and the
minor text changes you made to the non-url text are a violation of the
CR and CC terms it was released under.

4. Even if you satisfy the conditions in Section 3(a)(1), Section
3(a)(3) says, "If requested by the Licensor, You must remove any of
the information required by Section 3(a)(1)(A) to the extent
reasonably practicable" and I'd like to exercise that right and
explicitly request the removal of all the content (including URL's and
(re)modified text) that were already in `Julia.jl` and to not include
any content, or modify any content, that already appears in `Julia.jl`
in any renamed repo (especially awesome-julia) that is not a fork of
`Julia.jl` and retains the original name it was released under. It is
indeed very unfortunate that I am forced to do this because my
repeated requests to collaborate and contribute via PR's is being
ignored, you have not explained what stopped you from submitting a PR
for any (or all) missing links that you claim are your contributions.

Instead your response(s) in this thread and the commit message
(Updated IDE after copyright claims . Weird!!!
<eb9ac4e4c5e184cf8a9f94f74adbe8c8dae97fca>) make it clear that you
prefer to be snarky, dont respect me for bringing up the issue of CR
and CC violation, choose to not collaborate and prefer to keep a new
repo (which you are free to do sans any content from Julia.jl, now or
in future) that copies content already available.

Further references:
[1] The section "What do I need to do to get a copyright?" at
https://wiki.creativecommons.org/Frequently_Asked_Questions

Hope that explains my thoughts on the CR and CC license violations.
SVAKSHA ॥  http://about.me/svaksha  ॥

**Comment by svaksha** [comment]
On Fri, Jan 9, 2015 at 2:57 PM, Isaiah <notifications@github.com> wrote:
>
> I haven't read the links in question yet, but: could everyone please make an effort to stay on the main issue, which is the copyright question.
>
> I don't think it is the goal of the community to promote copyright violations if such exists.

Thank you for clarifying that Isaiah and thanks for alerting me to the
copyright violation. I missed mentioning that explicitly. TBH, I had
intended to fork the website repo and add to the docs when I chanced
upon the merged PR.

> At the same time, I would hope that we can give the benefit of the doubt and try to understand and educate on the requirements of each licensing scheme. This is a very important topic for the continued strength of open-source software, but it also takes time to understand the details. My own (very limited) understanding is that CC-BY-SA can be considered "like the GPL for documentation", and as such any direct copying means that the same license should be applied (as well as attribution provided).
>

I'm glad you think licensing issues are an important discussion worth
having. To clear any ambiguity, `Julia.jl` is under the CC-BY-NC-SA
(Attribution-NonCommercial-ShareAlike) != CC BY-SA
(Attribution-ShareAlike) as the latter allows commercial purposes, and
this is the “copyleft” free and open source software license you refer
to. Not true for CC-BY-NC-SA -4.0.

IANAL, and yet in my response I've tried to explain the copyright and
the cc-by-nc-sa-4.0 license violation as simply as possible, but would
love to hear from anyone with more experience with CC Licenses in FOSS
- this is the first time I've experienced this in FOSS so I am
learning as I go along. Based on the points raised in my earlier
replies, I would like to request the following from the Julia
community:

1. Formulate a clear policy regarding CC license violations and list
them somewhere where on the site or the codebase so that people are
aware of what to do in such situations and how to seek remedial
actions.

2. Secondly, I'd also like to request that the `awesome-julia` repo
not be listed on the Julia website as it violates the sharealike terms
under CC-BY-NC-SA-4.0 and Copyrights by including plagiarised content
from `Julia.jl` - Do grep the commit diffs vis-a-vis `Julia.jl` since
this discussion started and you will find a large number commits with
copied content with minor text changes. That cannot be called original
work.

It is disingenious to not notice that while he was easily able to
collaborate with other (male) owners of similar lists (viz.
awesome-php and awesome-python), he chooses to ignore my repeated
requests to collaborate on `Julia.jl`, choosing to break the CR and CC
license instead. Anyway, since `awesome-julia` is being listed on the
Julia website with a license and copyright violation, it forces me to
check each commit to his repo and file a Bug Report here whenever
there is a CR or licence violation in future. TBH, I would like to
avoid doing this, as its really unproductive and also wastes the
community time - I had to seek the advice and help of folks more
knowledgeable about these matters than me and I'd like to avoid having
to unnecessarily trouble them further. When it comes to FOSS there are
better ways to spend time and I prefer to go the positive route.

That said, ignoring the CC and CR violations is not the right way to
encourage new contributions to Foss - it goes against the spirit of
collaboration and FOSS ethos, so with that in mind I'd like to, once
again, ask @melvin0008 to contribute via PR's to `Julia.jl` and
respect the revocation of the CC rights in my earlier reply under
Section 3(a)(3) of the CC-BY-NC-SA-4.0.

Thanks, SVAKSHA ॥  http://about.me/svaksha ॥

**Comment by ViralBShah** [comment]
I have no disagreement with having one great collection, and I would prefer that it be curated.

Now, for my own understanding, If I make a copy of Julia.jl, then modify some parts, retain the original license, and credit the original author, mentioning my changes - isn't this something the CC license allows? Would I be considered engaging in plagiarism then?

https://creativecommons.org/licenses/by-nc-sa/4.0/

@melvin0008 has changed his license. He needs to credit the original source, and mention the changes (perhaps in his README.md). Is there anything else? I am removing awesome-julia temporarily until the licensing is properly done. I will reinstate the work when it is compliant, and over time some of his stuff can be merged into Julia.jl too.

However, I still request (and this request of mine has been consistently ignored) that the discussion be kept to the technicalities of licensing, and no attacks be made.

**Comment by melvin0008** [comment]
@svaksha The difference between Julia.jl and awesome-julia is Julia.jl concentrates on packages from various fields and awesome-julia concentrates more on the learning perspective . Most of my listing have been done using julialang.org as the primary resource. Others have been done using google. There was only one part i.e the IDE which I had copied. I removed that part yesterday and have written a new IDE list, if you compare it to yours you will clearly see the differences. 
I curated this IDE list after searching for the respected languages cited at [link](https://github.com/JuliaLang/julia#editor-and-terminal-setup).  

So technically currently I haven't copied anything from anywhere.


@viral Although as of yesterday, I changed the IDE part which was copied. I curated an entire new list as you can see. I curated this IDE list after searching for the respected languages cited at [link](https://github.com/JuliaLang/julia#editor-and-terminal-setup) . 
I have still given @svaksha [link](https://github.com/melvin0008/awesome-julia/blob/master/README.md#ide).

I have kept the conversation technical but would like to note that the constant jibes of me being a sexist are inappropriate.

**Comment by svaksha** [comment]
On Sat, Jan 10, 2015 at 4:38 AM, Viral B. Shah <notifications@github.com> wrote:
>
> I have no disagreement with having one great collection, and I would prefer that it be curated.
>
> Now, for my own understanding, If I make a copy of Julia.jl, then modify some parts, retain the original license, and credit the original author, mentioning my changes - isn't this something the CC license allows?

It does. In such situations, people automatically submit a PR (if they
dont one encourages them to do so) to the upstream repo as the
collective collaborations help everyone when they are shared alike.


> Would I be considered engaging in plagiarism then?
>

The plagiarism (which violates the Indian CR, which violates the
international CC license respectively) occurs when you change the repo
name and include content from Julia.jl. If you recall, my original BR
mentions how Stefan asked permission for using it as a package
reference, which I granted because he asked and I trusted him enough
to not violate the terms. Usually when people ask, goodwill is
automatically assumed.


>
> @melvin0008 has changed his license. He needs to credit the original source, and mention the changes (perhaps in his README.md). Is there anything else? I am removing awesome-julia temporarily until the licensing is properly done. I will reinstate the work when it is compliant, and over time some of his stuff can be merged into Julia.jl too.
>

Thank you Viral. A clear policy and a strong stand on CR and
CC/software licensing for the community will goo a long way in
preventing such violations in future.

Fwiw, I thought I had explained why inspite of the license change, the
Copyright and CC-BY-NC-SA-4.0 violation continues, but since you asked
:

1. Under By-nc-sa-4.0 you have to share-alike - that means, you can
fork, change and share `Julia.jl` while retaining the name of the
repo. When you create a new repo named `awesome-julia` (or any other
repo name), then copy over content, it breaks my CR and the CC license
it was released under which is valid internationally. In other words,
you cannot attribute content you didnt originally create, hence dont
own under the Copyright Act law. The finer nuances may not be clear or
obvious to readers, so to be explicit, I added the CR terms and
committed it yesterday. Also, JFTR, copyright is as per jurisdiction
and as per the Indian law, the creator owns the right which is defacto
automatically assigned to any material s/he originally created,
furthermore, unlike some western nations Indians cannot waive away CR
(which is yours even if you explicitly waived it away, 'cause the law
says so). IANAL, so please dont ask me to explain why or how this is
so.

2. As I mentioned earlier, any copied content from `Julia.jl` is a
violation and since I have exercised the rights provided under the
Section 3(a)(3) of the cc-by-nc-sa-4.0 and asked him to remove all the
copied content (whether attributed or not) from `awesome-julia`; he
will always be walking a fine line of a CR and CC licence violation if
he continues to maintain `awesome-julia`, irrespective of whether the
Julia website lists it.

3. Since we are aware of the violation I do know that it is not
productive usage of my (and the community) time to continue trailing
future violations. Hence, my suggestion that he (and anyone) else are
free to collaborate and contribute PR's for missing content to
`Julia.jl`; and to be explicitly clear, revoked the CC rights in the
case of `awesome-julia` and the Julia.jl fork he has on github (and
elsewhere) as I cannot speculate if this will happen in future, nor
want to waste everyone's time when the collaboration option exists in
FOSS. In case its not clear, given the current situation, I cannot
merge any of his PR's into `Julia.jl` while he continues to maintain
`awesome-julia` as I dont want to inadvertently break the CR or CC
license. After he deletes it, he is free to submit PR's to `Julia.jl`
which I'm happy to accept.

Hope this clears the air.

> However, I still request (and this request of mine has been consistently ignored) that the discussion be kept to the technicalities of licensing, and no attacks be made.
>

Again, please be specific about which comment in my BR is not civil -
without the specifics its pointless to accuse (me?) and sours the
discussion. If you are alluding to me asking Melvin a question, viz,
about why he broke the CR and CC license for my repo while being able
to contribute and collaborate with other devs on github; then, that is
a query based on a valid observation of an event (and action) that has
already occurred, and cant be (mis)interpreted as an attack. That
would be a logical fallacy. In toto, I'd just rather avoid this line
of conversation (which can quickly get unproductive) so please dont
accuse me if you are not willing to provide specifics.

SVAKSHA ॥  http://about.me/svaksha  ॥

**Comment by svaksha** [comment]
On Sat, Jan 10, 2015 at 7:00 AM, Melvin <notifications@github.com> wrote:
>
> @svaksha The difference between Julia.jl and awesome-julia is Julia.jl concentrates on packages from various fields and awesome-julia concentrates more on the learning perspective . Most of my listing have been done using julialang.org as the primary resource. Others have been done using google. There was only one part i.e the IDE which I had copied. I removed that part yesterday and have written a new IDE list, if you compare it to yours you will clearly see the differences.
> I curated this IDE list after searching for the respected languages cited at link.
>
> So technically currently I haven't copied anything from anywhere.

Melvin, If you wish to play the denial game, I am willing to contest
each commit content which violates the CR and CC license terms with
individual BR's. IMHO, that would be a complete waste of everyone's
time but since you are unwilling to delete the plagiarised content
copied from 'Julia.jl` and are asking `awesome-julia` be listed on the
site with the copied content, you leave me with no other option,
unless the Foss community can talk some sense into you. I would still
suggest that you not do this.

>
> @viral Although as of yesterday, I changed the IDE part which was copied. I curated an entire new list as you can see. I curated this IDE list after searching for the respected languages cited at link .
> I have still given @svaksha link.
>
> I have kept the conversation technical but would like to note that the constant jibes of me being a sexist are inappropriate.
>

I am going to ignore this as its not inappropriate to ask a question
based on your PR's to other repos.

- @svaksha.

**Comment by melvin0008** [comment]
@svaksha you can go ahead and look for copied commits .[I compiled a new IDE list yesterday] I can assure you everything has been found using google or information already existing on julialang.org. What content do you think is plagiarised? Please let me know. I will delete it.

**Comment by ihnorton** [comment]
@svaksha: regarding the extent of copyright, it is my understanding that facts are not copyrightable. As an example, Julia includes tables of mathematical data which are not subject to copyright even though they were almost certainly typed in from a book (at least originally). A list of Julia IDEs is not a copyrightable thing (IMHO, IANAL, etc.), but your  original *descriptors* and *arrangement* certainly are. There is some kind of gray area I don't understand in between "phone numbers" and "photocopy of the phonebook", but clearly the verbatim copying was on the wrong side and was unacceptable. As such, I support Viral's decision to remove the link.

(for the sake of completeness, the exception to the "arrangement" criterion would probably be alphabetical ordering, which is a fact of English)

Regarding collaboration, there are many examples I can think of where people do their own thing: Keno built Cxx.jl instead of collaborating within "my" Clang.jl (which was obviously the correct decision as Cxx.jl is superb). Doug Bates is building a new RCall even though there are 2-3 (plus?) others written already written at various levels of maturity. How many Julia autodiff libraries are there? And, don't get me started on testing frameworks (15?). I can't imagine open-source working in any other way than "you work on what you want to work on."

Regarding CC-BY-NC-SA (thanks for the correction), per the [CC FAQ] (https://wiki.creativecommons.org/Frequently_Asked_Questions#When_is_my_use_considered_an_adaptation.3F) it does not seem accurate to me to say that modifications/derivatives may only be made on a fork of Julia.jl. I could be wrong, and would appreciate the clarification if so, but I believe the CC-ND variant would provide stronger protections if that is your goal. Also see the section on [license revocability](https://wiki.creativecommons.org/Frequently_Asked_Questions#What_happens_if_the_author_decides_to_revoke_the_CC_license_to_material_I_am_using.3F). To the extent that I can understand the license (as per above), "awesome-julia" would qualify for the automatic reinstatement by adding the attribution (BY) and license (SA) -- both steps of which has been done, although Melvin has gone further and seems to have deleted all verbatim text in the IDE section and replaced it with his own descriptive text (I have not done a line-by-line comparison elsewhere).

Regarding the other point you made earlier:

```
1. Formulate a clear policy regarding CC license violations and list
them somewhere where on the site or the codebase so that people are
aware of what to do in such situations and how to seek remedial
actions.
```

This sounds fine. I have some wording in mind and will discuss in a separate pull request.

**Comment by svaksha** [comment]
On Sat, Jan 10, 2015 at 7:26 AM, Melvin <notifications@github.com> wrote:
>
> @svaksha you can go ahead and look for copied commits .[I compiled a new IDE list yesterday] I can assure you everything has been found using google or information already existing on julialang.org. What content do you think is plagiarised? Please let me know. I will delete it.
>

I will file the BR here later. Meanwhile, why this PR[1] when I
clearly mentioned about 30 minutes ago that given the current license
violation situation, I cannot merge your PR's into `Julia.jl` while
you maintain `awesome-julia` in parallel as I dont want to
inadvertently break the CR or CC license. Care to explain your motive
in doing this?

Core-devs, please step in and stop this. TBH, I'm finding this
attitude highly hostile and offensive as despite explaining multiple
times I am being cornered as if I reject his PR, he can then claim I
rejected the contribution after asking for it. This isnt how Foss
contributions should be encouraged.

[1] https://github.com/svaksha/Julia.jl/pull/19

SVAKSHA ॥  http://about.me/svaksha  ॥

**Comment by melvin0008** [comment]
@svaksha I'm not going to claim anything. 
And regarding the PR1 , my understanding is that if I take the initiative to add the links to IDE it shouldn't be a problem .

**Comment by melvin0008** [comment]
@svaksha  I have stated many times. My motivation for creating awesome-julia has been to let new developers with the set of resources to learn and start development to Julia. I made a mistake of copying the IDE section of Julia.jl . Consider it to be an amateur mistake. But I have corrected my mistake and changed the entire IDE part of awesome-julia. Julia.jl focuses on libraries and resources needed for specific fields. Mine is a overview and a great start for developers who are excited about Julia. All of the content I have curated has been referred from Julialang.org and simple google searches. You have ignored my kind request to kindly point out what you feel is plagiarized as of today. My view is for both the lists to exist in parallel and help the developers to get the required information , they are looking for. For me that's the spirit of Opensource and would be in the interest of propelling Julia as one of the top dynamic languages. 

Other than this, you have berated me and accused me of sexism which by all means is a stretch and an attempt towards defamation. This is clearly inappropriate.  

Please be cordial and point out issues you still feel are plagiarized. I would be happy to solve this issue once and for all.

Thanks.

**Comment by Keno** [comment]
Folks,
in order to keep this civilized, let's please review the facts.
If we disagree on the facts, we should probably fix that first, before moving on to issues of
law, because we'd be arguing from a different viewpoint.

I think we agree on this:

- @svaksha has the sole copyright to Julia.jl and has chosen to license it as CC-BY-NC-SA, the full license text of which is [here](http://creativecommons.org/licenses/by-nc-sa/4.0/legalcode)
- @melvin0008 has admitted to copying the list of IDEs from IJulia.jl. If done not in accordance with CC-BY-NC-SA, this constitutes copyright violation (not making any claim about whether this occurred yet).
- @melvin0008 originally chose the Apache license
- The Apache License is incompatible with the CC-BY-NC-SA
- Hence copying content directly from IJulia.jl constitute copyright infringement and breach of license

IANAL, but here is my understanding of the copyright situation. My view represents a background of American and European copyright. If this is different in India (as was indicated it might be in early discussion), please indicate so, so we may understand the subtleties involved.

- The CC-BY-NC-SA, only covers the content, exposition and organization of Julia.jl. In particular, the concept of a list of facts related to julia is not copyrightable.
- There could still be a copyright claim on the choice of categories (is this claim being made?)

Now, for the legal parts of the license itself. This is my interpretation from studying the license text. Again, please let me know if (either side) interprets this differently.

- Section 2.a.1.A, gives anybody the right to
```
**reproduce and Share** the Licensed Material, in whole or **in part**, **for NonCommercial purposes** only
```
- I think so far we are fine. The material was reproduced and shared in part for non-commercial purposes
- The portion of the license that was broken is 2.a.5.C, which states
```
No downstream restrictions. You may not offer or impose any additional or different terms or conditions on, or apply any Effective Technological Measures to, the Licensed Material if doing so restricts exercise of the Licensed Rights by any recipient of the Licensed Material.
```
As well as 3.b which states
```
ShareAlike.
In addition to the conditions in Section 3(a), if You Share Adapted Material You produce, the following conditions also apply.

The Adapter’s License You apply must be a Creative Commons license with the same License Elements, this version or later, or a BY-NC-SA Compatible License.
You must include the text of, or the URI or hyperlink to, the Adapter's License You apply. You may satisfy this condition in any reasonable manner based on the medium, means, and context in which You Share Adapted Material.
You may not offer or impose any additional or different terms or conditions on, or apply any Effective Technological Measures to, Adapted Material that restrict exercise of the rights granted under the Adapter's License You apply.
```
(this is the Share Alike part) as well as (this is the attribution part) 3.a
```
Attribution.

If You Share the Licensed Material (including in modified form), You must:

retain the following if it is supplied by the Licensor with the Licensed Material:
identification of the creator(s) of the Licensed Material and any others designated to receive attribution, in any reasonable manner requested by the Licensor (including by pseudonym if designated);
a copyright notice;
a notice that refers to this Public License;
a notice that refers to the disclaimer of warranties;
a URI or hyperlink to the Licensed Material to the extent reasonably practicable;
indicate if You modified the Licensed Material and retain an indication of any previous modifications; and
indicate the Licensed Material is licensed under this Public License, and include the text of, or the URI or hyperlink to, this Public License.
You may satisfy the conditions in Section 3(a)(1) in any reasonable manner based on the medium, means, and context in which You Share the Licensed Material. For example, it may be reasonable to satisfy the conditions by providing a URI or hyperlink to a resource that includes the required information.
If requested by the Licensor, You must remove any of the information required by Section 3(a)(1)(A) to the extent reasonably practicable.
```
- Hence the original copyright and license was violated

- However, the license was since changed, which alleviates the violation of 2.a.5.C and 3.b, so we are left with 3.a.

Now, there is a question of whether the given attribution is sufficient. From my reading of the license it probably isn't, as the license requires four explicit things to be part of the attribution.

- However, if the attribution were adjusted to be compliant, the license would automatically be reinstated under section 6(b)1.

So the two core questions are (which, @svaksha I would ask you to answer)
- Is there still any copyright violation in the current version of @melvin0008's project?
- If the attribution were corrected, would you still think there is any issue?

 @svaksha, I don't think anybody doubts the hard work, you've been putting in in maintaining this list. Personally, seeing a notification when you add one of my projects to the list always puts a smile on my face because somebody noticed :). I agree with your decision to reject the PR while this discussion is pending. I don't think anybody can fault you for that. I don't think anybody is trying to alienate anybody, but these are tricky issues.

Also, I don't think anybody here doubts the importance of copyright and licensing. We've had so many discussions where we couldn't use code because of it's license and we take the issue seriously. That said, licenses are legal documents and it is best to argue based on the verbiage of the license (I recognize you have done this).

I hope this post can serve as a basis of further discussion. I would encourage either side to pick out any specific bullets they disagree with.

**Comment by Keno** [comment]
@melvin0008 I understand your frustration, but in this matter I don't think it is relevant, as the license does not have a "good intention" clause. Also, I would encourage both parties to temporarily ignore the non-technical issues of the discussion (and not respond to them) until the legal question is resolved. If either side still wants to make a statement one that is done, by all means do (but keep it civilized), but let's resolve the legal issue first to avoid further resentment. If there's anything I can do to help out with this, please let me know.

**Comment by melvin0008** [comment]
@Keno I did make a copyright violation. But I have changed the entire content of the debated IDE part of awesome-julia. Doesn't that change all this?

**Comment by Keno** [comment]
@melvin0008 Yes, that's why I asked in one of my core questions
```
Is there still any copyright violation in the current version of @melvin0008's project?
```
If not, then the license question is indeed irrelevant.

**Comment by melvin0008** [comment]
@Keno  Okay Thanks.

**Comment by one-more-minute** [comment]
> Even if you satisfy the conditions in Section 3(a)(1), Section
3(a)(3) says, "If requested by the Licensor, You must remove any of
the information required by Section 3(a)(1)(A) to the extent
reasonably practicable" and I'd like to exercise that right and
explicitly request the removal of all the content [...]

To clarify this:
```
If requested by the Licensor, You must remove any of the information required by Section 3(a)(1)(A) to the extent reasonably practicable.
```
Section 3(a)(1)(A) specifically refers to the required attribution:
```
i. identification of the creator(s) of the Licensed Material and any others designated to receive attribution, in any reasonable manner requested by the Licensor (including by pseudonym if designated);
ii. a copyright notice;
iii. a notice that refers to this Public License;
iv. a notice that refers to the disclaimer of warranties;
v. a URI or hyperlink to the Licensed Material to the extent reasonably practicable;
```
In other words, the copyright holder may ask the adapter to remove the *attribution* which is otherwise legally required. This does not apply to the copyrighted content itself, however.

While it's polite to avoid reusing people's content if they request it, it's certainly not legally required by this license, as long as the attribution is in place.

Also, I don't see any terms that reference the possible motives of the adaptor, nor any licensing implications those motives would have.

**Comment by Keno** [comment]
Yes, @one-more-minute, I noticed that point as well. Thank you for clarifying.
@melvin0008 I of course don't want to tell you what to say or do, but I would wait until the legal issue is resolved.

**Comment by melvin0008** [comment]
Sure.

**Comment by one-more-minute** [comment]
Also, RE attribution I would point out 3.a.2:

> You may satisfy the conditions in Section 3(a)(1) in any reasonable manner based on the medium, means, and context in which You Share the Licensed Material. **For example, it may be reasonable to satisfy the conditions by providing a URI or hyperlink to a resource that includes the required information.**

Since the Julia.jl repository is itself a resource that includes the required information (the five bullets above), this suggests that simply linking to it is enough to satisfy the attribution requirement.

**Comment by melvin0008** [comment]
@one-more-minute  
```
Also, I don't think anybody here doubts the importance of copyright and licensing. We've had so many discussions where we couldn't use code because of it's license and we take the issue seriously. That said, licenses are legal documents and it is best to argue based on the verbiage of the license (I recognize you have done this).
```
Have I complied to all the rules? Or do I need to change something?

**Comment by one-more-minute** [comment]
You've attributed the Julia.jl repository in your readme, which from my reading of the license gives you the right to copy and paste as much of it as you want. Given that you have removed the copied content anyway, I would say that you've gone above and beyond what the license strictly requires and certainly don't need to do anything more.

However, it would of course be best to wait for @svaksha's input – we want to make sure everyone is happy with the outcome here.

**Comment by melvin0008** [comment]
Thanks a lot for simplifying all the legal jargons.

**Comment by nahratzah** [comment]
(Svaksha and I have been talking about this issue yesterday and I promised my input.  Unfortunately writing it took a bit longer than anticipated, some information has been covered already.)


On Plagiarism and Licensing
===========================
Taking Svaksha's IDE list and copying it into Melvin's repository, without attribution, is [Plagiarism](http://en.wikipedia.org/wiki/Plagiarism "Wikipedia Article on Plagiarism").  By omiting attribution, the list was passed off as Melvin's own work, instead of Svaksha's work.  I'm glad to see attribution is fixed.

The license on Svaksha's work detailed one way in how to do this right, in section 3a of the [license](http://creativecommons.org/licenses/by-nc-sa/4.0/legalcode "Creative Commons BY-NC-SA 4.0 License").  The attribution section is explicit on the information you have to provide.  I would argue the text 'Credits for curating the IDE List given to Svaksha' is insufficient based on that.  It suggests to me, that Svaksha did a pull request into your repository (or mailed a diff or similar), instead of this being part of a larger work Svaksha compiled.  It may even suggest she's actively involved in that section of your list, which she is not.

Another argument you could try to make, is that copying the list falls under [Fair Use](http://en.wikipedia.org/wiki/Fair_Use "Wikipedia Article on Fair Use").  Fair Use is a way of using the work, without following the license requirements or violating copyright.  Note that fair use still requires a citation reference (attribution).  Without this reference, it is still plagiarism.  Fair Use does not cover including part of a work into a larger work, so Fair Use does not apply in this instance.

Circling back to [Plagiarism](http://en.wikipedia.org/wiki/Plagiarism "Wikipedia Article on Plagiarism"), Plagiarism is a term originating from the Academic and Journalistic world.  Note that Plagiarism has nothing to do with the author of the original work complaining or not.  When you copy a text and don't attribute, irrespective of doing that intentional or accidental, the act is considered Plagiarism.  Plagiarists are usually cast out by their peers.  Back when I went to university, people commiting plagiarism were expelled, whereas people publishing scientific and journalistic works with plagiarism were expelled from their (scientific and journalistic) communities.

In most cases, edits to an existing work don't even create a new work.  An obvious example is fixing a typo.  Just because you changed a letter doesn't make it an original work.  The same goes for rewording.  For example, changing the text "Yoda, I talk like" to "I talk like Yoda" doesn't suddenly make it your original work.  Taking a single chapter (like the IDE section) and adding a lot of other text, may make the resulting text an original work, but won't make that copied chapter an original work.  Copying the IDE section and then rewording it puts awesome-julia clearly in the danger zone of a license and copyright violation.  Wether it is actually a violation, is up to a court of law, but I certainly wouldn't want to be in Melvin's position.

[Re ViralBSha](https://github.com/JuliaLang/julialang.github.com/issues/194#issuecomment-69321599 "Origin of following quote"):
> It is perfectly fine for someone to take pieces of other work and combine them in new and interesting ways.
> That is a fundamental basis of open source.
> We work hard on avoiding fragments, and often people are happy to collaborate.
> In some cases, the forks are meaningful and even invited.

You are right that combining work in new and interesting ways, is a pillar of open source.  Usually, this involves software.  However note that this ability usually comes with restrictions.  For instance, if I link against a GPL library, I must make my software GPL.  If I adapt someone's code, I must maintain the original copyright and license.  These requirements are spelled out in the license document.  The same holds for text.


Why Contacting JuliaLang
========================
Svaksha spent a lot of work on creating her list.  It involves finding projects, understanding what they are about, thinking of a one-line description and categorizing them.  Unattributed copying of the list removes the recognition she'll get for this work and instead directs it to Melvin.  The instance specifially addressed here, is the recognition and endorsement Melvin received from the JuliaLang project.  Not only does JuliaLang praise Melvin for the list (including it in your list of learning materials), by including it, it also legitimizes it.  For these reasons, Svaksha is right to bring it to the attention of JuliaLang.


On Sexism
=========
Svaksha has pointed out that Melvin choose to cooperate with aweomse-python and awesome-php lists, but choose not to do this with Julia.jl.  She also pointed out her experience with Indian men was bad, therefor prefering an open discussion instead.  This happens in [open source communities](https://modelviewculture.com/pieces/leaving-toxic-open-source-communities "Leaving Toxic Open Source Communities") and [this](http://juliepagano.tumblr.com/post/46206589124/my-experiences-in-tech-death-by-1000-paper-cuts "My experiences in tech: Death by 1000 paper cuts - Julie Pagano") [happens](http://www.catehuston.com/blog/2014/05/28/ways-to-undermine-women-whilst-being-nice/ "Your Guide to Undermining Women Whilst Being Nice - Accidentally In Code") in [tech](http://fortune.com/2014/10/02/women-leave-tech-culture/ "Why women leave tech: It's the culture, not because `math is hard'").

[Re ViralBSha](https://github.com/JuliaLang/julialang.github.com/issues/194#issuecomment-69321599 "Origin of following quote"):
> Also, there are far too many jibes against various types of people in your comment above - which are personal views and not helping the discussion here, and strictly not civil.

They're not views, but they're experiences.  They validate her reluctance to handle this quitly.  These experiences are also shared by a lot of people, making them not personal either.  She even pointed this out in the piece where you responded to her. :(  I don't see what is uncivil about it either, she calmly explained why she wanted a public discussion, included background information and even pointers on how to read up more on the subject.  If anything, she went above and beyond the minimal requirements for an answer and kept it short too.

[Re IDE Commit in Awesome-Julia](https://github.com/melvin0008/awesome-julia/commit/eb9ac4e4c5e184cf8a9f94f74adbe8c8dae97fca "Commit containing quoted message")
> Updated IDE after copyright claims . Weird!!!

The commit message reads as if Melvin doesn't care much about the issue.  Obviously the license problem is not on his priority list.  But by extension, he doesn't respect Svaksha either, otherwise her finding this a big deal would have made the license problem a big issue by proxy.  Instead, he refers to the whole thing as 'weird'.

It's non-trivial to explain this issue well.  From your point of view, this is an isolated incident.  Melvin is just someone who made some unfortunate mistakes.  From our viewpoint (Svaksha and I) this is part of a pattern, a systemic problem that turns up again and again.  Being ignored, being thought of as non-technical, being questioned on our technical or mathematical skills...  On top of that, many people aren't even aware they're doing it, it's [that deep ingrained in society](http://www.reddit.com/r/AskWomen/comments/2gk1su/what_subtle_sexist_things_do_men_do_that_they/ "Reddit - What (subtle) sexist things do men do that they think is 'normal' or are unaware of?").

I touched on this briefly in the 'Why Contacting JuliaLang' section, but I need to reiterate it here.  Men taking credit for the work a woman did happens. A lot.

From our point of view, Melvin looks like someone who'll happily collaborate with men, but won't with women, instead copying their content.  This may not even have been a conscious decision of him, but that doesn't mean it's not a sexist act.

**Comment by Keno** [comment]
Thanks @nahratzah for your comment. I will not touch on the sexism, because I do not feel like I'm in a position to comment. I recognize sexism exists in tech and that it is a significant problem. 
Regarding the other other two sections:

I agree with everything you said. Specific comments/followups inline.

> [ On Plagarism ]

I think you'll find widespread agreement in this community. A significant number of Julia contributors are academics and we take these issues seriously.

> I would argue the text 'Credits for curating the IDE List given to Svaksha' is insufficient based on that. 

Yes, as I mentioned I agree that the attribution was probably insufficient. Would you and @svaksha  agree with the verbiage `the list of IDEs was taken from (Julia.jl)[link] which is maintained by @svaksha and was licensed under the terms of the CC-BY-NC-SA`?

> so Fair Use does not apply in this instance.

I agree.

>  Copying the IDE section and then rewording it puts awesome-julia clearly in the danger zone of a license and copyright violation. Wether it is actually a violation, is up to a court of law, but I certainly wouldn't want to be in Melvin's position.

Yes. Copyright law is tricky and people sometimes need to be reminded of it. I hope (and think) @melvin0008 has learned from this experience.

> [Why Contacting JuliaLang]

Yes, I understand this and it makes sense. However, please understand that this puts us in a tricky position. We are not a court of law and thus cannot make legal judgements. There are other avenues for that including filing a DMCA claim with GitHub. We hope to make sure that the community we have built up is as friendly and welcoming as possible, and in this capacity we can try to resolve the issue (ideally by talking to both parties and educating where misinformation exists thus trying to resolve the conflict amicably). I hope the received response shows that we care about these issues.

As was discussed early, it would make sense to have a general policy on plagarism and license violation in the officially curated lists, but we should probably discuss that in a separate issue.

Ultimately, I would like @svaksha to comment on whether she still considers any of the material in @melvin0008's repository as it stands to be plagiarized. Afterwards, I think a discussion on collaboration (for which an issue on Julia.jl would probably be a more appropriate venue) can happen if both sides are still interested. We do like to see people join forces, but of course we can't require anybody to collaborate.

**Comment by melvin0008** [comment]
@nahratzah 

Regarding Plagiarism:
I have rewritten the debated IDE part of awesome-julia. This does not conform well with your example of yoda. I have compiled a new list of resources after googling them for each text editor mention in this [link](https://github.com/JuliaLang/julia#editor-and-terminal-setup). So the part of IDE list still being plagiarized is out of question. I have time and again agreed to having made a mistake. But recompilation of the IDE list rectifies my mistake.

On sexism:
Its really astonishing that you keep throwing that claim on me. It's better to comment on that when you start to know me. To be honest it is difficult to figure out the gender of the person without a display picture. Svaksha means beautiful eyed lord. So you berating me on sexism is not professional and hostile behaviour. So please stop misconstruing events and judge me.The defamation related to sexism is clearly unwanted and can be easily avoided. 

@svaksha:
Please let me know what content you still find to be plagiarized. I would be happy to remove it and would deeply regret the plagiarism caused.

@Keno I have definitely learnt a lot regarding Licenses and Plagiarism. I will never make such a mistake again.

**Comment by melvin0008** [comment]
@svaksha  Please reply so that we can get over this issue. 
Thanks.

**Comment by melvin0008** [comment]
Its been one week . Could we close this issue.

**Comment by ViralBShah** [comment]
I am closing this as I feel this is adequately resolved.

**Comment by svaksha** [comment]
@nahratzah, Thanks for the detailed response. Sorry about the slow response - been travelling with patchy internet access.

@Keno, Thank you for acknowledging the CC/CR violation. While I'm glad that the core-devs take this seriously enough, it would be nicer and clearer if there were a public policy outlining the do's and dont's - there have been some discussions in the past on the mailing list and usually the persons involved step back and stop. That isnt the case here, so to answer your question: 

> So the two core questions are (which, @svaksha I would ask you to answer)
>
> Is there still any copyright violation in the current version of @melvin0008's project?

YES, please see the commits linked below. 

>
> If the attribution were corrected, would you still think there is any issue?

The crux of the problem is that its not just one attribution, rather a number of commits (see below) that have been plagiarised over a short period into `awesome-julia`. We would not be having this discussion (nor would I need to file a BR) if his additions and changes were to the `Julia.jl` fork, as that would have preserved my commits and those of other collaborators. That isnt the case here.

I agree with @nahratzah who has already explained the copyright and CC license violations very nicely with the 'fair use' and the "yoda" example for plagiarism. Ofcourse, Melvin disagrees: "So the part of IDE list still being plagiarized is out of question" (https://github.com/JuliaLang/julialang.github.com/issues/194#issuecomment-69466087), which means he intends to continue copying stuff over if `awesome-julia` continues to exist. Thusfar, everyone I've asked  agrees its a CR/CC license violation and like you they suggested a DMCA takedown, including going public with this entire episode. 

Since the bug report was closed on Melvin's demand, it kills the discussion route and feels like silencing. Not at all what I expected. In this specific case, the reality is that Melvin didnt dare to break the license and plagiarise a man's work, so within the FOSS community, it sends the signal that its acceptable to steal a womans work. Feel free to disagree with this POV but that is my take-away from this experience which isnt very different from what other women have experienced. 

Before delving into the commits, I do want to address and describe why it takes a lot more work than "I got the link from the Julia website and via google". Julia being a new language is constantly in a state of flux and the easy part is listing packages listed in the official package list as they would have clear-cut descriptions of what the package does, the deps, tests, supported versions, etc... ala, say PyPI or CPAN or CRAN, where you can easily figure out what the package does and get back to work. BUT what of those packages (for lack of a better term) that are not on METADATA? And there are numerous such packages (erm...code repos?) that are not a part of the official list but I intentionally list these unregistered packages in `Julia.jl`, despite it not conforming to the standards - well, that is still being discussed and formulated by the community so I try not to be too rigid when listing unofficial information. 

While `Julia.jl` makes it relatively easy to figure out the package category, its not always true, so allow me to describe the effort that goes into classifying them by citing a hypothetical example of a package called "HMM.jl" :: HMM is an acronym used for "Hidden Markov models", "Heavy meromyosin" or it could be a library that tracks "Human malignant melanoma" which can have a parent category of Biochemistry, or Statistics. The category and classification is easy when the package owner has described or provided some information on the repo either via a description or the README has some description. For the most part, that isnt the case as the package naming standard schema is a WIP and evolving as the discussions within the community grows. 

I dont use a large majority of the packages, but still obsess over figuring out what the package does when the repo has no README or description, and if the package name is a cryptic acronym (like 'HMM') it can be anything - a lib used in Math, Biology, Statistics or something else entirely. Assuming its the former, I have to start reading the code to figure out the classification of the parent category. If the HMM package was written with a focus on bioinformatics AND/OR another type of "science" research, it can be again sub-classified under say "Cancer Research", that is further sub-categorized under say, genetics, epidemiology, cell biology, etc.., respectively. If the HMM package has a Statistics/Math focus, it has to be listed as per the sub-cat on the top level page. Sometimes the information is available if the devel wrote tests (not really, it would then be in the official metadata list, though not necessarily), or via comments within the code, the function names, etc which I co-relate with a search engine result or a paper I read, or the work the dev does - Example, their profile may state they are a researcher working in a Cancer lab or a Statistics lab and co-relating that with similar 'category' packages in their repo page helps me guessimate what the package does (or not - I make mistakes too). Some developers may not agree with the category chosen and they do write asking me to relist their package under a different CAT than the one I chose (thanks), or some may ask me to remove the listing as its not meant for public consumption, or just write to say thanks for listing it. When the latter happens, I assume they are ok with the categorization :-)

Roughly, that is the most simplistic way to describe the effort that goes into the categorization and while the category tagging may seem like a trivial obsession over very little (the package description), therein lies the key difference. Boot-strapping a new language and getting it accepted into an existing production system is hard and depends on a large library ecosystem support that ensures that people find the library the developer spent hours writing (/testing/developing), use it and improve it and add more features and contribute upstream if they are so inclined. I'm happy that `Julia.jl` is the tiny catalyst that helps accelerate that process and since I'm not into social networking, the entire credit for that goes to those community folks who found `Julia.jl`, tweeted about it, starred it, forked it and contributed to it, linked it on the website and on wikipedia/researchgate, their notebooks, recommended it to others via blog posts/newsletters/SO/reddit/Quora, et al. A BIG Thank you :-D !


# List of commits that plagiarise `Julia.jl` content.

I spent quite some time and took the trouble of going through each commit as it would be needed for the DMCA legal action and its important enough to be listed here for future reference. I've linked the commits datewise (some days have multiple commits) and the corresponding markdown file in `Julia.jl` where the information was organized. 

### Commits on Jan 11, 2015 
============================

1. Commit: https://github.com/melvin0008/awesome-julia/commit/24f454ec24aa846f76695c8aac30858d67d34745#diff-04c6e90faac2675aa89e2176d2eec7d8R41 
* Avik's JavaCall.jl lib is already listed in `Julia.jl` in the API.md page.

2. Commit : https://github.com/melvin0008/awesome-julia/commit/a5b66418ddcced25208ef412b8b5ce9e2641c357
* The [Julia-vim](https://github.com/JuliaLang/julia-vim) editor has already been listed in `Julia.jl` in the Utilities.md page.

3. Commit: https://github.com/melvin0008/awesome-julia/commit/b31c9b63fc3b011cc94a3afff0bd96361caf5dc5 
* The section "+## Calling other Languages" was created and 2 library links, namely, ObjectiveC.jl and PyJulia that were already listed in the API.md page in `Julia.jl` were copied over to this section. Here is a git log for reference: 
.git/logs/refs/heads/master:82db4048b1c2d29ebd728b717225469194f49c4d 22687abfa177ff51067532669da66972ae14bf03 SVAKSHA <svaksha@gmail.com> 1414040213 +0530	commit: @one-more-minute : ObjectiveC.jl
API.md:- [ObjectiveC.jl](https://github.com/one-more-minute/ObjectiveC.jl) :: A Julia module for Objective-C.
API.md:- [PyJulia](https://github.com/jakebolewski/pyjulia) :: python interface to julia.


### Commits on Jan 10, 2015 
=============================
List of links copied from `Julia.jl`: 

 4. https://github.com/melvin0008/awesome-julia/commit/19fed8dfb14f1c45afdfdafd8bb50e61b779a53b, already present in Resources.md:- [Rosetta-Julia](https://github.com/quinnj/Rosetta-Julia) :: Rosetta Code tasks implemented in the Julia language.

 5. https://github.com/melvin0008/awesome-julia/commit/db7b1e4901d1412ef7c796836f55b0171d9f07d2, is already listed in the page QA.md:- [FactCheck.jl](https://github.com/zachallaun/FactCheck.jl) :: Midje-like testing framework written for Julia.

 6. https://github.com/melvin0008/awesome-julia/commit/a19a83932243ef60f518281acf94d38d9b08022b, present in Resources.md:- [julia-tutorial](https://github.com/JuliaLang/julia-tutorial) :: Julia Tutorial from MIT IAP 2013.

 7. https://github.com/melvin0008/awesome-julia/commit/74b529a3e201cf53e567b9a4a162597e3e65a456
Already present in the pages
* Build-Automation.md:- [Download and install Julia on various Operating Systems](http://julialang.org/downloads/)
* Web-Server.md:   - [Morsel.jl](https://github.com/JuliaLang/Morsel.jl) :: is a Sintra-like micro framework for declaring routes and handling requests, built over HttpServer.jl and Meddle.jl.

 8. https://github.com/melvin0008/awesome-julia/commit/78e9d42eb5e4ab11a5896dd6452435478b8d8d1c
List of groups and repos already listed in the following markdown pages in `Julia.jl`
* NEWS.md: Join the [http://julialang.org/community/](mailing list), or head over to [#julia on irc.freenode.net](http://webchat.freenode.net/?channels=julia) for a live chat.
* Statistics.md:- [JuliaStats group](https://github.com/JuliaStats/) and their [home page](https://github.com/JuliaStats/JuliaStats.github.io) on github.
* Statistics.md:- [JuliaOpt](http://juliaopt.org/) :: is an umbrella group for Julia-based optimization-related projects, with its own [julia-opt mailing list](https://groups.google.com/forum/#!forum/julia-opt) at Google groups.
* Statistics.md:- [Julia Quant group on Github](https://github.com/JuliaQuant)
* Astronomy.md:* [JuliaAstro group on Github](https://JuliaAstro.github.io)
* Chemistry.md:- [JuliaQuantum](http://juliaquantum.github.io/), is the umbrella Org for Quantum chemistry, Lindblad dynamics, Quantum tensor network, information and computation, etc...
* AI.md:- [JuliaSparse](https://github.com/JuliaSparse) for development of interfaces to sparse matrix solvers such as sequential MUMPS, SuperLU, perhaps Pastix and fill-reducing permutation software such as Metis and Scotch.
* Mathematics.md:##### YAJO :: http://www.juliadiff.org/ || https://github.com/JuliaDiff/
* Parallel-Computing.md:- [JuliaGPU](https://github.com/JuliaGPU) organization on GitHub.
* Web-Server.md:#### YAJO - JuliaWeb
Web-Server.md:- [JuliaWeb](https://github.com/JuliaWeb)
* Biology.md:- [BioJulia group](https://github.com/BioJulia) on github, and the GoogleGroups mailing list:: [https://groups.google.com/forum/?fromgroups#!forum/biojulia-dev](https://groups.google.com/forum/?fromgroups#!forum/biojulia-dev)
* Utilities.md:- [JuliaBox](http://www.juliabox.org/) :: Jupyter/IJulia sandboxed by Docker containers that allows you to run Julia in the browser. [Source code](https://github.com/JuliaLang/JuliaBox) and a [collection of example notebooks](https://github.com/shashi/IJuliaNotebooks) for juliabox.org.
* Morsel.jl is already listed in #7 above.

9. Commit: https://github.com/melvin0008/awesome-julia/commit/6f79def2c654cfc1e713d7399f63a6cdbfb076e0
* Utilities.md:- [Julia-Studio](http://forio.com/products/julia-studio/) :: Julia Studio by Forio.com and [Tutorials for Julia-Studio](http://forio.com/products/julia-studio/tutorials/) and its [source on github](https://github.com/forio/julia-tutorials).
* Utilities.md:- [Jupyter](http://jupyter.org) 'nee [IJulia.jl](https://github.com/JuliaLang/IJulia.jl) :: Julia kernel and magics for IPython.

10. Commit : https://github.com/melvin0008/awesome-julia/commit/0bec335f001f0a0d74091d090f58213be833d5ff
* Utilities.md:- [Julia-Vim](https://github.com/JuliaLang/julia-vim).
* Utilities.md:- [emacs-ess-julia.el](https://github.com/emacs-ess/ESS/blob/master/lisp/ess-julia.el) :: ESS support for julia language, includes font-lock, indentation, sending code to sub-process, interactive documentation, imenu, completion and eldoc. 

### Commits on Jan 9, 2015 
=============================

11. Commit: https://github.com/melvin0008/awesome-julia/commit/76a54d4108dffe873df101376ba435fbbb6ef98d
Available in the `Julia.jl` pages:
* Publications.md:- [Julia language research publications](http://julialang.org/publications/).
* Resources.md:- [Julia-cheatsheet.pdf](http://math.mit.edu/~stevenj/Julia-cheatsheet.pdf) :: StevenJ's Julia cheatsheet.

12. https://github.com/melvin0008/awesome-julia/commit/dba82057855a11640cc2cedfc4be54f728707282
Planet was also listed in the page:
* Resources.md:- The [Julia Planet](http://www.juliabloggers.com/) aggregates community blogs on Julia and the content is also aggregated into [Twitter](https://twitter.com/juliabloggers). If you'd like to contribute your content to Julia Bloggers, [submit your RSS/Atom feed link here](http://www.juliabloggers.com/julia-bloggers-submit-rss-feed/). 

13. https://github.com/melvin0008/awesome-julia/commit/cec07afd71171ad4df37a58c164c17e85ca95c69
The Julia Standard Library and Cheatsheet were already listed in :
* Resources.md:- [Julia-cheatsheet.pdf](http://math.mit.edu/~stevenj/Julia-cheatsheet.pdf) :: StevenJ's Julia cheatsheet.
* Build-Automation.md:- [The Julia Standard Library](http://docs.julialang.org/en/latest/stdlib/)

14. https://github.com/melvin0008/awesome-julia/commit/eb9ac4e4c5e184cf8a9f94f74adbe8c8dae97fca
To repeat what @nahratzah said, https://github.com/JuliaLang/julialang.github.com/issues/194#issuecomment-69462349, Copying sections and links already in the 'Julia.jl' repo and then rewording it puts 'awesome-julia' clearly in the danger zone of a license and copyright violation. All of the above and below also fall in the same category of a CC/CR violation.

15. Commit: https://github.com/melvin0008/awesome-julia/commit/b120e565580377bd9b1f96e69240f6fe1ab79936, where he repeats yet another link already listed in the page Resources.md:- ([A Julia Tutorial](http://julialang.org https://github.com/JuliaX/JuliaTutorial) 

### Commits on Jan 8, 2015 
==========================

16. Commit : https://github.com/melvin0008/awesome-julia/commit/7ce5d09040931a27927c5e5689b3c16391183a8b,  already listed in page Build-Automation.md:- [Download and install Julia on various Operating Systems](http://julialang.org/downloads/)

### Commits on Jan 6, 2015 
==========================

17. https://github.com/melvin0008/awesome-julia/commit/ad829e1022811c24749814d4a99a1b5ed92dc60a, where the entire IDE subsection was plagiarised, then reworded and hence according to Melvin "So the part of IDE list still being plagiarized is out of question." (from: https://github.com/JuliaLang/julialang.github.com/issues/194#issuecomment-69466087)

### Commits on Jan 5, 2015 
===========================

18. https://github.com/melvin0008/awesome-julia/commit/016f17964508cca08078caaaeed6fb1c4ba17923, already in NEWS.md:- [Stackoverflow](http://stackoverflow.com/questions/tagged/julia-lang)

19. https://github.com/melvin0008/awesome-julia/commit/016f17964508cca08078caaaeed6fb1c4ba17923, again to be found in Resources.md:- The [Julia Planet](http://www.juliabloggers.com/) aggregates community blogs on Julia and the content is also aggregated into [Twitter](https://twitter.com/juliabloggers). If you'd like to contribute your content to Julia Bloggers, [submit your RSS/Atom feed link here](http://www.juliabloggers.com/julia-bloggers-submit-rss-feed/).

20. Reddit link from NEWS.md:- [Reddit](http://www.reddit.com/r/Julia/)

### Commits on Jan 4, 2015 
===========================

21. https://github.com/melvin0008/awesome-julia/commit/01830e50c31674bf24a0db39f5053825f94c25d0 lists the Bangalore Julia User Group, already mentioned in NEWS.md:- [Bangalore Julia User Group](http://www.meetup.com/Bangalore-JULIA-User-Group/) and the Resources.md page links to the JuliaTokyo second meetup slides(http://juliatokyo.connpass.com/event/8010/presentation/).

### Commits on Jan 2, 2015  
=============================

22. The commit https://github.com/melvin0008/awesome-julia/commit/b45efcc636b7e9b4be539ef76ac4fbece3ee81ce, is yet another verbatim copy of the North American meetup sub-section, already listed in the markdown page NEWS.md: https://github.com/svaksha/Julia.jl/blob/master/NEWS.md#julia-user-groups
(The Bay Area Julia Users, Cambridge Area Julia Users Network (C.A.J.U.N.), Triangle Julia Users - NC, NYC Julia User Group, Chicago Julia Meetup group, and Ottawa JUG)

23. Commit https://github.com/melvin0008/awesome-julia/commit/ad8b554079541fd1cfe89837d2f5b6c8007805d8, links to the [Julia User Groups](http://julia.meetup.com) which is already linked on the same page.

### Commits on Jan 1, 2015 

24. https://github.com/melvin0008/awesome-julia/commit/4814faaa97feeffb1a1efb575c3fe2059a778aca 
JuliaBox is already listed in Utilities.md:- [JuliaBox](http://www.juliabox.org/) :: Jupyter/IJulia sandboxed by Docker containers that allows you to run Julia in the browser. [Source code](https://github.com/JuliaLang/JuliaBox) and a [collection of example notebooks](https://github.com/shashi/IJuliaNotebooks) for juliabox.org.

25. https://github.com/melvin0008/awesome-julia/commit/ef073bc0b5a8aec032a86762755a468a38a0f88b 
JuliaOpt ML is already listed in Statistics.md:- [JuliaOpt](http://juliaopt.org/) :: is an umbrella group for Julia-based optimization-related projects, with its own [julia-opt mailing list](https://groups.google.com/forum/#!forum/julia-opt) at Google groups. 

### Commits on Dec 31, 2014 
=============================

26. https://github.com/melvin0008/awesome-julia/commit/107787df4902bdac1013fe0dea34e8aea6537800 <== https://github.com/JuliaLang, is listed in the README.md and for the other copied links, please see the Jan 10, 2015 commit listed above as # 8. https://github.com/melvin0008/awesome-julia/commit/78e9d42eb5e4ab11a5896dd6452435478b8d8d1c 

27. https://github.com/melvin0008/awesome-julia/commit/1042c302c6ba1a3f17c6d0dcbe6b07a3165e287d, links to http://pkg.julialang.org/ which is already listed in the README.md:For Base packages, check if the package you seek is listed in their [built-in package manager](https://github.com/JuliaLang/METADATA.jl) on github, or check METADATA for [registered Julia packages](http://pkg.julialang.org/),...

28. https://github.com/melvin0008/awesome-julia/commit/305c0fa41c24b6d79f8ba7817d82f39724175be8
chrisvoncsefalvay's "learn-julia-the-hard-way" and stevenj's "Julia-cheatsheet.pdf" are already listed in the Resources.md page in `Julia.jl`.

29. https://github.com/melvin0008/awesome-julia/commit/bb52b098220e2a8eca82f3b1189530ace67c6851
The SciPy 2014 tutorial is also listed in the Resources.md page.

30. https://github.com/melvin0008/awesome-julia/commit/7e473191bf9bf67b11b880d431249a9ece2af850,
Forio's Julia-Studio tutorials: The "http://forio.com/products/julia-studio/tutorials/" redirects to "http://forio.com/labs/julia-studio/tutorials/"
Utilities.md:- [Julia-Studio](http://forio.com/products/julia-studio/) :: Julia Studio by Forio.com and [Tutorials for Julia-Studio](http://forio.com/products/julia-studio/tutorials/) and its [source on github](https://github.com/forio/julia-tutorials).

@Keno, hope this list of more than 30 commits tells you why its hard to ignore the copyright and CC violation - by continuing to copy and plagiarise one link at a time into `awesome-julia`, @melvin0008 is not creating anything new or original, rather destroying all the work and contributions that went into making `Julia.jl` what it is today. 


@melvin0008, your reply [on sexism to @nahratzah](https://github.com/JuliaLang/julialang.github.com/issues/194#issuecomment-69466087) crosses the line when you ask her to get to know you. Atleast a dozen women (of all age groups) who I polled said the same thing - "when you start to know me" is used by guys trying to hit on them. Seriously, dont. Dont be THAT guy!
Moving on, the cultural smackdown on @nahratzah was not necessary: "To be honest it is difficult to figure out the gender of the person without a display picture. Svaksha means beautiful eyed lord." <== Your assumption is wrong. To respect anothers work and collaborate online (or in real life), you dont need personal and irrelevant information (picture or gender). My gender isnt a secret but by terming it masculine ("lord") you certainly display a lack of knowledge of the Buddhist "theravadin" philosophy and Sanskrit grammar (hint: nighantu/nirukta).
Suggesting that women conform to the male perspective of online identity denies women (and minorities) their freedom to choose how they represent themselves online. [Wikipedia](https://en.wikipedia.org/wiki/Nymwars), GeekFeminism wiki and Skud (http://geekfeminism.org/2010/06/10/hacker-news-and-pseudonymity/) have some interesting writeups that make a good read if one wants to understand the perspective of _half the world's populace_.

**Comment by StefanKarpinski** [comment]
We've drafted a community standards document here:

http://julialang.org/community/standards/

It's not linked from the front page yet – that's the next step.

@svaksha: I didn't go through all of the links in your post, but I did go through a significant sampling of them – 15 or so. It seems that, aside from [the commit that copies the list of IDEs, including descriptions](https://github.com/melvin0008/awesome-julia/commit/ad829e1022811c24749814d4a99a1b5ed92dc60a), which @melvin0008 has acknowldged and removed, they do not share copyrightable text with your repository. Many of the links are the same, but links are facts, which cannot be copyrighted and don't fall under the scope of plagiarism either. Given that the copyrightable text about IDEs has been removed and @melvin0008 now links to Julia.jl (thereby qualifying to use the material under its creative commons license), do you agree that there is no longer any copyright violation occurring here?

@melvin0008, although lists of links (like tables of facts) are not copyrightable, I do think that it would be nice if you added at the top of the README, after the phrase "Inspired by awesome-php.", a sentence along the lines of "Many of these resources were originally collected and curated by @svashka and others – thank you for the hard work." Even if not legally necessary, as @svaksha has pointed out, curating such lists is a lot of work, so it seems appropriate to give as much credit as possible. You might also want to include a sentence about your motivation for making a different list – what perspective does your collection bring to the issue of curating Julia resources?

**Comment by melvin0008** [comment]
@StefanKarpinski I have changed the Readme as per what you said.
I have also stated my [motivation](https://github.com/melvin0008/awesome-julia/blob/master/README.md#motivation) for creating awesome-julia .

Thanks

**Comment by StefanKarpinski** [comment]
Thank you, @melvin0008. I've opened a separate issue for finalizing and adding links to the community standards document: https://github.com/JuliaLang/julialang.github.com/issues/200.

**Comment by melvin0008** [comment]
@svaksha I have removed the Readme. Any sane person will admit that there was no plagirism in the last version of awesome-julia. Your claims and long comments didn't have substance. Time and again I have tried to solve this issue cordially but that's something which didn't happen. But something tells me you will never stop pestering me. To save mine and other Devs precious time I would like to close this issue once and for all. Please educate yourself on how to talk to people. Class one value education book would be a great start . Thanks.

**Comment by melvin0008** [comment]
Regret posting the previous comment. I have deleted it .

**Comment by ihnorton** [comment]
The repo maintainers reiterate that we do not condone and will not tolerate personal attacks in any forum. Please forgive past mistakes and please exercise restraint in all comments.

This issue was [opened](https://github.com/JuliaLang/julialang.github.com/issues/194#issue-53851172) to address the following:

(1) a copyright violation and related link from JuliaLang/* resources. This violation has been acknowledged and the following corrective action was taken:
- we have removed the link to `awesome-julia` from the Julia homepage.

(2) codification of community standards with respect to copyright/attribution. To this end:
- a draft has been written and links are in progress (#200).

We feel that the specific issues raised above have been addressed as much as possible on this issue tracker. The comment area remains open, in case there are any further, actionable items related to the above issues.

Please be aware that this is not the appropriate forum for general copyright disputes, nor are any maintainers able or competent to adjudicate such issues.

We expect everyone to follow the [community standards](http://julialang.org/community/standards/) in any future comments and refrain from personally-directed comments.
